### PR TITLE
DOC: Adding utilities to the documentation tree.

### PR DESCRIPTION
**Pull request by hhslepicka**
DOC: Improved docstrings at colors and iconfont.

After merged this closes https://github.com/slaclab/pydm/issues/279 .

### CODE REVIEW

File: pydm/utilities/iconfont.py [hunk 1]
```diff
@@ -77,6 +77,23 @@ def font(self, size):
...
+        """
+        Retrieve the icon given a name and color.
+
+        Parameters
+        ----------
+        name : str
+            The Icon string identifier.
+            Icon strings can be found at: https://fontawesome.com/icons?d=gallery
+
+        color : QColor
```
Comment (teddyrendahl) [inline pydm/utilities/iconfont.py]: This is optional

File: pydm/utilities/colors.py [hunk 2]
```diff
@@ -12,6 +12,21 @@
...
 
 def svg_color_from_hex(hex_string, hex_on_fail=False):
+    """
+    Returns the SVG color based on its HEX equivalent.
+
+    Parameters
+    ----------
+    hex_string: str
+        The color code in hex. E.g. `#000000`
+    hex_on_fail: bool
```
Comment (teddyrendahl) [inline pydm/utilities/colors.py]: Thiis optional

File: pydm/utilities/colors.py [hunk 3]
```diff
@@ -12,6 +12,21 @@
 
 
 def svg_color_from_hex(hex_string, hex_on_fail=False):
+    """
+    Returns the SVG color based on its HEX equivalent.
+
+    Parameters
+    ----------
+    hex_string: str
+        The color code in hex. E.g. `#000000`
```
Comment (teddyrendahl) [inline pydm/utilities/colors.py]: Inline literals needs double `"``"`

File: pydm/utilities/colors.py [hunk 4]
```diff
@@ -21,6 +36,19 @@ def svg_color_from_hex(hex_string, hex_on_fail=False):
 
 
 def hex_from_svg_color(color_string):
+    """
+    Returns the HEX color based on its SVG equivalent.
+
+    Parameters
+    ----------
+    color_string: str
+        The SVG color string. E.g. `black`
```
Comment (teddyrendahl) [inline pydm/utilities/colors.py]: Inline literals needs double "``"

File: pydm/utilities/iconfont.py [hunk 5]
```diff
@@ -77,6 +77,23 @@ def font(self, size):
...
 
     def icon(self, name, color=None):
+        """
+        Retrieve the icon given a name and color.
+
+        Parameters
+        ----------
+        name : str
+            The Icon string identifier.
+            Icon strings can be found at: https://fontawesome.com/icons?d=gallery
```
Comment (teddyrendahl) [inline pydm/utilities/iconfont.py]: 👍

File: docs/source/conf.py [hunk 6]
```diff
@@ -38,9 +38,11 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.viewcode',
+    'sphinx.ext.githubpages',
```
Comment (teddyrendahl) [inline docs/source/conf.py]: Just curious. What does this do? Make the `.nojekyll` file?

Comment (hhslepicka) [reply docs/source/conf.py]: Yes! Exactly! 👍

**Comment by hhslepicka** [comment]
Thank you @teddyrendahl for the review. I addressed the comments on the latest commit.
It is ready for a new review round.

**Pull request merged**